runtime/stream.h

```diff
@@ -216,6 +217,6 @@ strmGetCurrFileNum(strm_t *pStrm) {
 /* prototypes */
 PROTOTYPEObjClassInit(strm);
 rsRetVal strmMultiFileSeek(strm_t *pThis, int fileNum, off64_t offs, off64_t *bytesDel);
-rsRetVal strmReadMultiLine(strm_t *pThis, cstr_t **ppCStr, regex_t *preg, sbool bEscapeLF);
+rsRetVal strmReadMultiLine(strm_t *pThis, cstr_t **ppCStr, regex_t *preg, sbool bEscapeLF, off64_t *offset);
```

Comment:
Some time (years) ago, we tried to modularize everything, so that thrid parties could make use of it. However, this hasn't really worked out. First of all, nobody really used it. Secondly, it gets very complicated in regard to version mixes. So there is no need to preserve these interfaces. In fact, as a very low priorty TODO item I have a refactoring of this to something more simple and efficient (regular dynamic linking).